bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_gg-fusion_b-associatedproduction/run2Template_cards/template_HToZATo2L2B_500p00_300p00_20p00_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8/HToZATo2L2B_500p00_300p00_20p00_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8_run_card.dat

```diff
@@ -0,0 +1,191 @@
+# Reweight variables for scale dependence and PDF uncertainty          *
+#***********************************************************************
+ 1.0, 2.0, 0.5 = rw_rscale ! muR factors to be included by reweighting
+ 1.0, 2.0, 0.5 = rw_fscale ! muF factors to be included by reweighting
+ True = reweight_scale ! Reweight to get scale variation using the 
+            ! rw_rscale and rw_fscale factors. Should be a list of 
+            ! booleans of equal length to dynamical_scale_choice to
+            ! specify for which choice to include scale dependence.
+ -1.0  = rw_fscale_down ! lower bound for fact scale variations
+ -1.0  = rw_rscale_down ! lower bound for ren scale variations
+ -1.0  = rw_fscale_up   ! upper bound for fact scale variations
+ -1.0  = rw_rscale_up   ! upper bound for ren scale variations
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+ True = store_rwgt_info ! Store info for reweighting in LHE file
+#***********************************************************************
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+ 0        = ickkw
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma). Determines which resonances are      *
+# written in the LHE event file                                        *
+#***********************************************************************
+ 15.0  = bwcutoff
+#***********************************************************************
+# Cuts on the jets. Jet clustering is performed by FastJet.            *
+#  - When matching to a parton shower, these generation cuts should be *
+#    considerably softer than the analysis cuts.                       *
+#  - More specific cuts can be specified in SubProcesses/cuts.f        *
+#***********************************************************************
+  1.0  = jetalgo   ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+  1.0  = jetradius ! The radius parameter for the jet algorithm
+ 10.0  = ptj       ! Min jet transverse momentum
```

Comment:
For your analysis, in your SR are you going to require nbjets>=2 ? If not (==> nbjets=1 also considered as SR), I won't give cuts to 10 GeV. This cut will only reduce sensitivity. Is this intended?

Reply:
Yes, my signal region is nbjets>=2 ( resolved region) or at least 1 fatbjet with 2 subjets inside (boosted region). 
We are also looking only for jets with `pt > = 20 GeV ` in the analysis. 
So having a cut  `10.0  = ptj       ! Min jet transverse momentum` doesn't consider to be loose enough? 
I thought 10 GeV will increase sensitivity actually by not generating events we know that we are gonna cut anyway at the analysis level.

Reply:
It is indeed loose enough when you require nbjet>=2 only.
My concern was that there might be a loss when you also include nbjet=1 SR and ptj=10 might take away some sensitivity. But if the analysis requires at least 2, it is okay.

Reply:
But I am using  `  4 = maxjetflavor`  in the run_card. So that cut won't be applied on the bjets. 
If you set `maxjetflavor = 4 ` then the cut on the b quark and the other jets are different. For example for the pt cut, the b are cut following "ptb"  variable (in my card  ptb= 0) while the other following "ptj" variable.

If you set `maxjetflavor =5` then the b quark and the other jet are cut in the same way. In this case both are cut following "ptj" variable.

Reply:
aaah my bad geez ok. wasn't an issue in the first place